src/components/modal/_modal.scss

```diff
@@ -10,7 +10,7 @@
     left: 50%;
     transform: translateX(-50%) translateY(-50%);
     box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.06);
-    overflow: auto;
+    overflow: visible;
```

Comment:
![image](https://github.com/City-of-Helsinki/mvj-ui/assets/83913239/b1e5ce62-100e-46b2-b161-1177f5f8115f)

Reply:
This change makes it easier to go through the search results. The overflow prop applies to all modals, but I haven't come up with any problem that might come from this.